src/main/java/gui/controllers/TimerController.java

```diff
@@ -0,0 +1,71 @@
+package gui.controllers;
+
+import java.util.concurrent.TimeUnit;
+import javafx.animation.AnimationTimer;
+
+/**
+ * TimerController class for controlling GUI elements.
+ */
+public class TimerController {
+
```

Comment:
waarom extend dit geen controller

Reply:
Omdat het geen controller is

Reply:
waarom heet het dan controller?

Reply:
Omdat ik geen betere naam wist, weet jij een betere naam?

Reply:
uhm

Reply:
ff denken hoor

Reply:
TimerController class for controlling GUI elements.